### DISCUSSION TITLE
Action more visible?

### DISCUSSION THREAD

**Discussion by ekandreas** [filamentphp/filament#5269] [Help]
We are currently using Laravel Nova instead of Filament Tables.
When doing some POC with tables we wonder if it is possible to gain a more visible action button or menu?
In Nova there is a dropdown very clear to the user as UX.

Is it possible to change the three dot icon at least to something bigger or more intuitive?

**Comment by danharrin** [comment]
Here's how it looks in v3

<img width="685" alt="Screenshot 2022-12-18 at 13 25 27" src="https://user-images.githubusercontent.com/41773797/208300859-597fc835-8269-4779-b108-bc3c327f9625.png">

**Reply by ekandreas** [reply]
Great! ❤️
